packages/get-uri/src/file.ts

```diff
@@ -55,8 +55,7 @@ export const file: GetUriProtocol<FileOptions> = async (
 
 		// `fs.ReadStream` takes care of calling `fs.close()` on the
 		// fd after it's done reading
-		// @ts-expect-error `@types/node` doesn't allow `null` as file path :/
-		const rs = createReadStream(null, {
+		const rs = createReadStream(filepath, {
```

Comment:
Is this change related to the PR? I'm curious if passing in the file path will open a new FD or not.

Reply:
Yes, this change was needed as unit tests were failing in Node 14: https://github.com/TooTallNate/proxy-agents/actions/runs/11353091855/job/31595561091

You'd have thought the `@ts-expect-error` comment would have silenced that TypeError.  

Providing the filepath as the first argument of createReadStream made the unit tests pass and meant I could remove the `@ts-expect-error` comment.

Passing the filepath to createReadStream when you already have a file descriptor explicitly set in the options does not open a new file descriptor. It simply uses the one provided:

https://nodejs.org/docs/latest-v20.x/api/fs.html#:~:text=If%20fd%20is%20specified%2C%20ReadStream%20will%20ignore%20the%20path%20argument%20and%20will%20use%20the%20specified%20file%20descriptor.